src/app/modules/registration/components/set-location-in-map/set-location-in-map.component.ts

```diff
@@ -148,9 +150,9 @@ export class SetLocationInMapComponent implements OnInit, OnDestroy {
           icon: locationMarkerIcon,
         });
       } else {
-        const lastView = await this.mapService.mapView$.pipe(take(1)).toPromise();
-        if (lastView) {
-          this.locationMarker = L.marker(lastView.center, {
+        const homePageMapView = await firstValueFrom(this.mapService.mapView$);
```

Comment:
Kanskje vi bare skal kalle dette `mapView`, siden det ikke trenger å komme fra homePage nødvendigvis, feks hvis man starter appen med url-parametere rett til sett lokasjon. Har ikke testa det, men jeg antar det går fint.

Reply:
Navnet var valgt for å skille den fra `this.mapView`, så jeg synes `mapView` blir litt forvirrende. Men jeg ser poenget. Hva med `initialMapView` e.l.?

Reply:
Navnet var valgt for å skille den fra `this.mapView`, så jeg synes `mapView` blir litt forvirrende. Men jeg ser poenget. Hva med `initialMapView` e.l.?